doc/releases/changelog-0.25.0.md

```diff
@@ -758,6 +758,11 @@ of operators.
 * Calling `qml.equal` with nested operators now raises a NotImplementedError.
   [(#2877)](https://github.com/PennyLaneAI/pennylane/pull/2877)
 
+* Returning `qml.sample()` or `qml.counts()` with other non-commuting measurements of observables
```

Comment:
Observables are non-commuting instead of the measurements themselves.
```suggestion
* Returning `qml.sample()` or `qml.counts()` with other measurements of non-commuting observables
```